qe-on-duty-reporter/qe_on_duty/reporters/daily_reporter.py

```
@@ -0,0 +1,283 @@
+
+            if len(issues) > 10:
+                sections.append(f"\n  *...and {len(issues) - 10} more bugs*")
+
+        return "\n\n".join(sections)
+
+    def _generate_footer(self) -> str:
+        """Generate report footer with repository listing and workflow filter config."""
+        categorized = self.config.get_categorized_repositories()
+        all_repos = self.config.get_all_repositories()
+
+        lines = [f"## Report Configuration\n"]
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)